### ISSUE TITLE
[BUG] GridSearchCV Warning: Non-Finite Test Scores for SVR Model

### ISSUE THREAD

**Issue by shristirwt** [rohitinu6/Stock-Price-Prediction#201]
### Has this bug been raised before?

- [X] I have checked "open" AND "closed" issues and this is not a duplicate

### Description

I am encountering a warning during the hyperparameter tuning of a Support Vector Regression (SVR) model using GridSearchCV in scikit-learn. The warning states: "One or more of the test scores are non-finite: [nan nan nan nan nan nan]."
Another warning occured when attempting to define a Keras Sequential model. The warning indicates that the model is improperly configured by passing the input_shape directly to the Dense layers rather than using an Input layer at the beginning of the model.

![Screenshot 2024-10-28 230852](https://github.com/user-attachments/assets/1baf14b0-288f-4656-82d2-1261196f0c99)
![Screenshot 2024-10-28 231629](https://github.com/user-attachments/assets/2be9a1d3-cb5d-49fe-9fdf-e8b621c6f2c9)
![Screenshot 2024-10-28 231822](https://github.com/user-attachments/assets/6c44f7c3-f9f3-453c-b522-512d7c31f4ba)

### Steps to Reproduce

Include any relevant details like:

- What page you were on...        stock_market(complete).ipynb
- What you were trying to do...         I was executing the code when I encountered the warning
- What went wrong...       It displayed the above mentioned warnings

### Do you want to work on this issue?

Yes

### If "yes" to the above, please explain how you would technically implement this.

I will switch the scoring metric to a regression-appropriate metric and for the second warning I will use Input layer
